lldb/test/API/functionalities/data-formatter/data-formatter-stl/generic/string_view/main.cpp

```diff
@@ -95,7 +95,6 @@ int main() {
   std::string_view *null_str = nullptr;
 
   std::string hello = "Hellooo ";
-  std::string_view oops = hello + "World\n";
```

Comment:
This tested implementation detail. For me, `oops` was `"\0ellooo World\n"`, so I removed this.